UpdateSvcutilCorePackage.ps1

```diff
@@ -0,0 +1,122 @@
+$logfile = Join-Path $PSScriptRoot 'UpdateSvcutilCorePackage.log'
+"Log file for ModifySvcutilCorePackage............."  | Out-File $logfile
+
+if ($args.Length -ne 1)
+{
+    "Invalid number of input parameters" | Out-File $logfile -Append
+    exit;
+}
+
+$path = $args[0]
+if (![IO.Directory]::Exists($path))
+{
+    "The path " + $path + " not exist" | Out-File $logfile -Append
+    exit
+}
+
+$target = Get-Childitem �Path $path | Where-Object Name -like svcutilcore*nupkg | Select-Object -First 1
+
```

Comment:
The dash before Path is an "en dash" and not a hypen like you intended. This is unicode character U+2013.